### ISSUE TITLE
Move the "Paper lookup" section

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1225]
The "Paper lookup" section on the front page is quite prominent and it's not obvious what it's for.  Perhaps we could move it lower on the page and change to a better title?  Maybe add one sentence about what it does?

**Comment by ValWood** [comment]
I agree, I made a comment about the  "Paper Lookup"  instruction in another ticket, but I can't find it....

**Comment by ValWood** [comment]
Question

This leaves
About community curation
Canto curation makes your work visible in PomBase and many other databases.

Should we lose that too?

Is there any way to make the title line more 'encouraging' instead?

**Comment by mah11** [comment]
> Perhaps we could move it lower on the page and change to a better title?

Yes, I think it would work to swap "First time curator?" and what's now called "Paper lookup".

> Maybe add one sentence about what it does?

Hmm, to parallel "First time curator?", we could use something chatty like "Already using Canto? Find your papers here".

**Comment by mah11** [comment]
> I agree, I made a comment about the "Paper Lookup" instruction in another ticket

only in your head, I'm afraid ...

**Comment by ValWood** [comment]
> Hmm, to parallel "First time curator?", we could use something chatty like "Already using Canto? Find your papers here".

yep...

> only in your head, I'm afraid ...

yep...

**Comment by mah11** [comment]
> Should we lose [about community curation] too?

my https://github.com/pombase/canto/issues/1204#issuecomment-242322908 is relevant (no strong pref, but keep or lose the lot; don't keep just part of it)

> Is there any way to make the title line more 'encouraging' instead?

I'm not convinced this page even needs a subtitle. It's a part of the page that doesn't get a lot of attention (thaaaanks, banner ads!), so we're not going to sink or swim based on what's there. If you do really really want one, how about "Contribute your data to PomBase" or "Add your work to PomBase"?

> "Increase the visibility of your publications in PomBase and other databases"

yuck. way too wordy

**Comment by ValWood** [comment]
"Add your work to PomBase"...nice and simple and not scary

**Comment by kimrutherford** [comment]
I've added "Search for papers you have previously curated:" under the "Paper lookup" heading.  Is that good enough?

**Comment by ValWood** [comment]
Will be...new tickets if anything to change
